javascript/javascript1/week2/student.js

```diff
@@ -0,0 +1,31 @@
+const class07Students = ["a", "b"];
+
+function addStudentToClass(studentName) {
+    if (studentName === "") {
```

Comment:
Check long empty strings also. Use [trim](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/String/Trim)
 for that.